src/base.php

```diff
@@ -295,7 +295,7 @@ protected static function requireFile( $fileName, $className, $prefix )
         {
             $array = require( "$autoloadDir$fileName" );
 
-            if ( is_array( $array) && array_key_exists( $className, $array ) )
+            if ( is_array( $array ) && array_key_exists( $className, $array ) )
```

Comment:
Having rules for the spaces between parens and $things willl find many hidden gems everywhere in all components, I already did a rough check over and its leading to a lot of standardization.